homeassistant/components/hue/sensor_base.py

```diff
@@ -160,7 +161,7 @@ def async_update_items(self):
             self._component_add_entities[platform](to_add[platform])
 
 
-class GenericHueSensor(GenericHueDevice, entity.Entity):
+class GenericHueSensor(GenericHueDevice, SensorEntity):
```

Comment:
The inheritence will be conceptually incorrect for hue now, since its binary sensor inherits from the general ZLL sensor class below. It's probably better to just have the sensor platform entity class inherit from SensorEntity.